Support Fast XML Parser v4.x

sgerrand opened the pull request:
💁 These changes update the version of `fast-xml-parser` to 4.2.4 to mitigate the following vulnerabilities:
- [CWE-1321](https://cwe.mitre.org/data/definitions/1321.html)
- [CWE-1333](https://cwe.mitre.org/data/definitions/1333.html)

Commit:
Bump fast-xml-parser to v4.2.4

This mitigates the following vulnerabilities:
- [CWE-1321](https://cwe.mitre.org/data/definitions/1321.html)
- [CWE-1333](https://cwe.mitre.org/data/definitions/1333.html)

Commit:
Update syntax for XML parser

This syntax changed between versions 3 and 4 of `fast-xml-parser`.

sgerrand commented:
👋 @kamilkisiela: are you able to review these changes?

kamilkisiela commented on packages/core/src/utils.ts:
```diff
@@ -1,10 +1,13 @@
-import xmlParser from "fast-xml-parser";
 import { TimeReport, Test } from "./types";
 
+const {XMLParser} = require("fast-xml-parser");
```
```suggestion
import {XMLParser} from "fast-xml-parser";
```

sgerrand replied:
Good catch, updating now.

sgerrand replied:
Updated in 8351d34. ♻️

Commit:
Update syntax from suggestion in code review

As suggested by @kamilkisiela in code review, thanks Kamil!

https://github.com/kamilkisiela/split-tests/pull/13#pullrequestreview-1485974370

sgerrand commented:
Thanks! Are you planning to release a new version with these changes?
